packages/mobile/src/components/drawers/RemoveDownloadedCollectionDrawer.tsx

```diff
@@ -11,11 +12,10 @@ const { getCollection } = cacheCollectionsSelectors
 
 const messages = {
   header: 'Are You Sure?',
-  description: (isAlbum: boolean) =>
-    `Are you sure you want to remove this ${
-      isAlbum ? 'album' : 'playlist'
-    } from your device?`,
-  confirm: 'Remove Downloaded Playlist'
+  description: (contentType: string) =>
```

Comment:
Nit: I usually type these with the union like 'album' | 'playlist' but this is fine too